astropy/modeling/functional_models.py

```diff
@@ -732,7 +732,7 @@ class Ellipse2D(Fittable2DModel):
     y_0 = Parameter()
     a = Parameter()
     b = Parameter()
-    theta = Parameter()
+    theta = Parameter(default=0)
```

Comment:
Will do. Along those lines I would modify Gaussian2D to default the `x_stddev` and `y_stddev` unless there's a good reason for that requirement. It's pretty explicit.